src/java/org/apache/cassandra/db/streaming/CassandraCompressedStreamReader.java

```diff
@@ -97,7 +97,7 @@ public SSTableMultiWriter read(DataInputPlus inputPlus) throws IOException
                 {
                     writePartition(deserializer, writer);
                     // when compressed, report total bytes of compressed chunks read since remoteFile.size is the sum of chunks transferred
-                    session.progress(filename, ProgressInfo.Direction.IN, cis.chunkBytesRead(), totalSize);
+                    session.progress(filename + '-' + fileSeqNum, ProgressInfo.Direction.IN, cis.chunkBytesRead(), totalSize);
```

Comment:
This is needed here because without it, the output in netstats look like this, which is wrong:

```
Repair ec3d7e50-d667-11ea-b3d0-8f5940ba25d7
    /127.0.0.1
        Receiving 3 files, 1890453 bytes total. Already received 2 files (66.67%), 528363 bytes total (53.09%)
            netstats_test/test_table 528363/528363 bytes (100%) received from idx:0/127.0.0.1
```

New output looks like this:

```
Repair ec3d7e50-d667-11ea-b3d0-8f5940ba25d7
    /127.0.0.1
        Receiving 3 files, 1890453 bytes total. Already received 2 files (66.67%), 1003673 bytes total (53.09%)
            netstats_test/test_table-1 528363/528363 bytes (100%) received from idx:0/127.0.0.1
            netstats_test/test_table-0 221727/221727 bytes (100%) received from idx:0/127.0.0.1
            netstats_test/test_table-2 253583/529137 bytes (47%) received from idx:0/127.0.0.1
```

The problem with the original code is that `netstats_test/test_table` is periodically put into `SessionInfo#updateProgress` but since we are putting there same "file name", it is replaced so there is ever just one entry in that map.

"file name" is not completely clear here, it is "logical file name" because it is not fully known yet where that will be be actually saved, the original "file name" does not make sense anyway ... I am appeding the sequence number just for the sake of having it unique.

Reply:
When writer is `org.apache.cassandra.io.sstable.format.RangeAwareSSTableWriter` getFileName is

```
return String.join("/", cfs.keyspace.getName(), cfs.getTableName());
```

else it is

```
return descriptor.filenameFor(Component.DATA);
```

But when entire file streaming is used, then the reader is `CassandraEntireSSTableStreamReader`  which uses `session.progress(writer.descriptor.filenameFor(component), ProgressInfo.Direction.IN, length, length);`


So when `RangeAwareSSTableWriter` is used we can't use a better name as it writes to different sstables